### PR TITLE
Pulsar fix for tools that use chromosome len files

### PR DESCRIPTION
**Pull request by gregvonkuster**
(Please replace this header with a description of your pull request. Please include *BOTH* what you did and why you made the changes. The "why" may simply be citing a relevant Galaxy issue.)
(If fixing a bug, please add any relevant error or traceback)
(For UI components, it is recommended to include screenshots or screencasts)

## How to test the changes?
(Select all options that apply)
- [ ] I've included appropriate [automated tests](https://docs.galaxyproject.org/en/latest/dev/writing_tests.html).
- [x] This is a refactoring of components with existing test coverage.
- [ ] Instructions for manual testing are as follows:
  1. [add testing steps and prerequisites here if you didn't write automated tests covering all your changes]

## License
- [x] I agree to license these contributions under [Galaxy's current license](https://github.com/galaxyproject/galaxy/blob/dev/LICENSE.txt).
- [x] I agree to allow the Galaxy committers to license these and all my past contributions to the core galaxy codebase under the [MIT license](https://opensource.org/licenses/MIT). If this condition is an issue, uncheck and just let us know why with an e-mail to galaxy-committers@lists.galaxyproject.org.

### CODE REVIEW

**Comment by gregvonkuster** [comment]
This fixes tools like bedtools https://github.com/galaxyproject/tools-iuc/blob/master/tools/bedtools/macros.xml#L90 so that the paths can be properly set for the remote server.

File: lib/galaxy/tools/wrappers.py [hunk 1]
```diff
@@ -51,7 +51,7 @@
 # field names and all such fields are assumed to be paths. This is to allow
 # remote ComputeEnvironments (such as one used by Pulsar) determine what values to
 # rewrite or transfer...
-PATH_ATTRIBUTES = ["path"]
+PATH_ATTRIBUTES = ["len_path", "path"]
```
Comment (mvdbeek) [inline lib/galaxy/tools/wrappers.py:54]: Can you also fix the .log file typo on line 50 please ?

**Review by mvdbeek — APPROVED** [review]

File: lib/galaxy/tools/wrappers.py [hunk 2]
```diff
@@ -47,11 +47,11 @@
 
 log = logging.getLogger(__name__)
 
-# Fields in .log files corresponding to paths, must have one of the following
+# Fields in tool config files corresponding to paths, must have one of the following
```
Comment (mvdbeek) [inline lib/galaxy/tools/wrappers.py]:
```suggestion
# Fields in tool config files corresponding to paths (e.g .loc or .len), must have one of the following
```

**Comment by mvdbeek** [comment]
Thanks @gregvonkuster!

**Pull request merged**